RobinHoudini/RHoudini.rdef

```diff
@@ -0,0 +1,180 @@
+	$"FF0F3F3F3F3F0000003F3F002020202025253F3F3F3F3F3F3F3F3F3F3F1700FF"
+	$"FF0F3F3F3F000000003F3F00000000003F3F3F3F3F3F3F3F3F3F3F3F3F1700FF"
+	$"FF0F3F3F0000003F003F3F003F3F3F3F3F3F3F3F3F3F3F3F3F3F3F3F3F1700FF"
+	$"FF0F3F3F00003F3F3F3F3F003F3F3F3F3F3F3F3F3F3F3F3F3F3F3F3F3F1700FF"
+	$"FF0F3F3F003F3F3F3F3F3F003F3F3F3F3F3F3F3F3F3F3F3F3F3F3F3F3F1700FF"
+	$"FF0F3F3F3F003F3F3F3F3F003F3F3F3F3F3F3F3F3F3F3F3F3F3F3F3F3F1700FF"
+	$"FF0F3F3F3F3F3F3F3F3F3F0017171717171717171717171717171717171700FF"
+	$"FF000000000000000000000000000000000000000000000000000000000000FF"
+};
+
+resource(0, "BEOS:M:application/x-vnd.be-directory") #'MICN' array {
+	$"FFFFFFFFFF0000FFFFFFFFFFFFFFFFFF"
+	$"FFFFFFFF0083830000FFFF0000FFFFFF"
+	$"FFFFFF000000AA83830000D9D90000FF"
+	$"FFFFFF002A2A0000AA83838383D9D900"
+	$"0000FF002A2A5DFA0000AA8383838300"
+	$"00D900002A2A5DFA51520000AA838300"
+	$"00AAD9D900005DFA51525E5E00AA8300"
+	$"FF008383D9D9000051525E5E00AA8300"
+	$"FF0083838383D9D900005E5E00AA8300"
+	$"FFFF008383838383D9D9005E00AA00FF"
+	$"FFFF00AAAA8383838383005E00AA00FF"
+	$"FFFFFF0000AAAA838383830000AA00FF"
+	$"FFFFFFFFFF0000AAAA83830000AA00FF"
+	$"FFFFFFFFFFFFFF0000AAAAAA00AA0010"
+	$"FFFFFFFFFFFFFFFFFF0000AA00000010"
+	$"FFFFFFFFFFFFFFFFFFFFFF00000010FF"
+};
+
+resource(1, "BEOS:M:application/x-vnd.microsoft-internetexplorershortcut") #'MICN' array {
+	$"FFFFFFFFFFFFFFFFFFFFFF5F5F5F5FFF"
+	$"FFFFFFFFFFFF5F5F5F5F5F5FFFFFFF5F"
+	$"FFFFFFFF5F5FB2B2B2B2935E5FFFFFB2"
+	$"FFFFFFD35F8D675F5F935E935E5FFFB2"
+	$"FFFFD3D48E185F5FD3D35F724C5E5FFF"
+	$"FFFFD48EFDB25F5FFFFFD393464C5FFF"
+	$"FFD389D9B2B25FFFFFFFFFD34C725E5F"
+	$"FF0E62ADB3B2B2B2B293935E5E5E5E5F"
+	$"0E6283B3B4B25F5F5F5F5F5F5F5F5F5F"
+	$"0E83B2D3B4B25FFFFFFFFFFFFFFFFFFF"
+	$"0B89B3D4B45FB2FFFFFFFFD3B2B2B27E"
+	$"0BB3DC25D3B4B25FFFFFD3B2B2B2B27E"
+	$"0BD3FFD5D4D3B4B25FB2B2B2B2B27EFF"
+	$"0ED3FFFF0202D3B4B45FB2B27E7EFFFF"
+	$"FFD37EFFFFDC0224D3B47E7EFFFFFFFF"
+	$"FFFFD3D3D3FFFFFFFFFFFFFFFFFFFFFF"
+};
+
+resource(2, "BEOS:M:application/x-vnd.ms-ie-url") #'MICN' array {
+	$"FFFFFF0F0F0F0F0F0F0F0F0FFFFFFFFF"
+	$"FFFFFF0F3F3F3F3F3F3F3F170FFFFFFF"
+	$"FFFFFF0F3F3F3F3F3F3F3F173F0FFFFF"
+	$"FFFFFF0F3F3F3F20203F3F00000000FF"
+	$"FFFFFF25202020253F203F3F3F1700FF"
+	$"FFFF25B1FA202020253F3F3F3F1700FF"
+	$"FF25B1FA203F3F2520253F3F3F1700FF"
+	$"FFB1FA202020202020253F3F3F1700FF"
+	$"B1172520253F3F3F3F3F3F3F3F1700FF"
+	$"0F0F0F0F0F0F002520253F3F3F1700FF"
+	$"0F3F3F3F3F3F0020253F3F3F3F1700FF"
+	$"0F3F3F00003F000F3F3F3F3F3F1700FF"
+	$"0F3F000F003F003F3F3F3F3F3F1700FF"
+	$"0F3F003F3F3F001717171717171700FF"
+	$"0F3F3F3F3F3F000000000000000000FF"
+	$"00000000000000FFFFFFFFFFFFFFFFFF"
+};
+
+resource file_types message;
+
+resource app_signature "application/x-vnd.kirilla-robinhoudini";
+
+resource app_version {
+	major  = 1,
+	middle = 0,
+	minor  = 0,
+
+	variety = B_APPV_FINAL,
+	internal = 0,
+
+	short_info = "Robin Houdini",
+	long_info = "Robin Houdini"
+};
+
+resource app_flags B_SINGLE_LAUNCH;
```

Comment:
Please put all of these at the top of the file (and delete file_types, it's useless)